Meta/Azure/Setup.yml

```diff
@@ -7,20 +7,17 @@ steps:
 
   - ${{ if eq(parameters.os, 'Serenity') }}:
     - script: |
-        sudo add-apt-repository ppa:canonical-server/server-backports
-        sudo add-apt-repository ppa:ubuntu-toolchain-r/test
         wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | sudo apt-key add -
-        sudo add-apt-repository 'deb http://apt.llvm.org/focal/ llvm-toolchain-focal-14 main'
+        sudo add-apt-repository 'deb http://apt.llvm.org/jammy/ llvm-toolchain-jammy-14 main'
         sudo apt-get update
         sudo apt-get install clang-format-14 ccache e2fsprogs gcc-11 g++-11 libstdc++-11-dev libmpfr-dev libmpc-dev ninja-build qemu-utils qemu-system-i386 unzip lld
       displayName: 'Install Dependencies'
 
   - ${{ if eq(parameters.os, 'Linux') }}:
     - script: |
         sudo apt-get purge -y clang-11 clang-12 gcc-10
-        sudo add-apt-repository ppa:ubuntu-toolchain-r/test
         wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | sudo apt-key add -
-        sudo add-apt-repository 'deb http://apt.llvm.org/focal/ llvm-toolchain-focal-13 main'
+        sudo add-apt-repository 'deb http://apt.llvm.org/jammy/ llvm-toolchain-jammy-13 main'
         sudo apt-get update
         sudo apt-get install ccache gcc-11 g++-11 clang-13 libstdc++-11-dev ninja-build unzip
```

Comment:
(Not related to this PR, just caught my eye)
@BertalanD Should Lagom CI be using LLVM 14 at this point?

Reply:
Sure, why not?

Ubuntu 22.04 ships LLVM 14 by default, but only version 10 and 11 are pre-installed on the VMs. I can open a feature request later today.